### PR TITLE
kinder: update the usage of build command

### PR DESCRIPTION
**Pull request by chansuke**
Add detailed usage of `build` command.

### CODE REVIEW

File: kinder/cmd/kinder/build/build.go [hunk 1]
```diff
@@ -26,11 +26,17 @@ import (
...
-		// TODO(bentheelder): more detailed usage
+		Args:  cobra.NoArgs,
 		Use:   "build",
 		Short: "Build one of [base-image, node-image-variant]",
-		Long:  "Build the base node image (base-image) or node image variants (node-image-variant)",
+		Long: `
+		The 'build' command is used to create images necessary for a Kubernetes cluster created with kinder.
+		
+		It has two primary subcommands:
+
```
Comment (neolit123) [inline kinder/cmd/kinder/build/build.go:36]:
```suggestion
```

File: kinder/cmd/kinder/build/build.go [hunk 2]
```diff
@@ -26,11 +26,17 @@ import (
...
 		Use:   "build",
 		Short: "Build one of [base-image, node-image-variant]",
-		Long:  "Build the base node image (base-image) or node image variants (node-image-variant)",
+		Long: `
+		The 'build' command is used to create images necessary for a Kubernetes cluster created with kinder.
+		
+		It has two primary subcommands:
+		
+		1. 'base-image': This command is used to build the base image for nodes in the cluster. The base image includes all the necessary dependencies and configurations required for Kubernetes nodes.
+
```
Comment (neolit123) [inline kinder/cmd/kinder/build/build.go:38]:
```suggestion
```

File: kinder/cmd/kinder/build/build.go [hunk 3]
```diff
@@ -26,11 +26,17 @@ import (
...
+		Args:  cobra.NoArgs,
 		Use:   "build",
 		Short: "Build one of [base-image, node-image-variant]",
-		Long:  "Build the base node image (base-image) or node image variants (node-image-variant)",
+		Long: `
+		The 'build' command is used to create images necessary for a Kubernetes cluster created with kinder.
+		
+		It has two primary subcommands:
+		
+		1. 'base-image': This command is used to build the base image for nodes in the cluster. The base image includes all the necessary dependencies and configurations required for Kubernetes nodes.
```
Comment (neolit123) [inline kinder/cmd/kinder/build/build.go:37]:
```suggestion
		1. 'base-image': This command is used to build the base image for nodes in the cluster.
		The base image includes all the necessary dependencies and configurations required for Kubernetes nodes.
```

File: kinder/cmd/kinder/build/build.go [hunk 4]
```diff
@@ -26,11 +26,17 @@ import (
...
+		
+		2. 'node-image-variant': This command is used to build different variants of the node images based on the base image. These variants may include different Kubernetes versions, CNI plugins, or any other variations as needed for testing different Kubernetes features and behaviors.`,
```
Comment (neolit123) [inline kinder/cmd/kinder/build/build.go:39]:
```suggestion
		2. 'node-image-variant': This command is used to build different variants of the node images based
		on the base image. These variants may include different Kubernetes versions, CNI plugins,
		or any other variations as needed for testing different Kubernetes features and behaviors.,
```

File: kinder/cmd/kinder/build/build.go [hunk 5]
```diff
@@ -26,11 +26,17 @@ import (
...
 func NewCommand() *cobra.Command {
 	cmd := &cobra.Command{
-		Args: cobra.NoArgs,
-		// TODO(bentheelder): more detailed usage
+		Args:  cobra.NoArgs,
 		Use:   "build",
 		Short: "Build one of [base-image, node-image-variant]",
-		Long:  "Build the base node image (base-image) or node image variants (node-image-variant)",
+		Long: `
+		The 'build' command is used to create images necessary for a Kubernetes cluster created with kinder.
```
Comment (neolit123) [inline kinder/cmd/kinder/build/build.go:33]:
```suggestion
		The 'build' command is used for creating images necessary for a Kubernetes cluster created with kinder.
```

File: kinder/cmd/kinder/build/build.go [hunk 6]
```diff
@@ -26,11 +26,18 @@ import (
 // NewCommand returns a new cobra.Command for building
 func NewCommand() *cobra.Command {
 	cmd := &cobra.Command{
-		Args: cobra.NoArgs,
-		// TODO(bentheelder): more detailed usage
+		Args:  cobra.NoArgs,
 		Use:   "build",
 		Short: "Build one of [base-image, node-image-variant]",
-		Long:  "Build the base node image (base-image) or node image variants (node-image-variant)",
+		Long: `
```
Comment (neolit123) [inline kinder/cmd/kinder/build/build.go:32]: text should start on this line

**Review by neolit123 — COMMENTED** [review]
```
[*] Verifying whitespace...
Verifying trailing whitespace...
Found trailing whitespace in the follow files:
./cmd/kinder/build/build.go:[3](https://prow.k8s.io/view/gs/kubernetes-jenkins/pr-logs/pull/kubeadm/2972/pull-kubeadm-kinder-verify/1728217824842747904#1:build-log.txt%3A3)[4](https://prow.k8s.io/view/gs/kubernetes-jenkins/pr-logs/pull/kubeadm/2972/pull-kubeadm-kinder-verify/1728217824842747904#1:build-log.txt%3A4):		
Verifying new lines at end of files...
Found whitespace errors!
```

CI is failing. make sure there are no left-over spaces or new lines.

File: kinder/cmd/kinder/build/build.go [hunk 7]
```diff
@@ -26,11 +26,16 @@ import (
...
 		Use:   "build",
 		Short: "Build one of [base-image, node-image-variant]",
-		Long:  "Build the base node image (base-image) or node image variants (node-image-variant)",
+		Long: "The 'build' command is used for creating images necessary for a Kubernetes cluster created with kinder. " +
+			"It has two primary subcommands: " +
+			"1. 'base-image': This command is used to build the base image for nodes in the cluster. " +
+			"The base image includes all the necessary dependencies and configurations required for Kubernetes nodes. " +
+			"2. 'node-image-variant': This command is used to build different variants of the node images based" +
+			"on the base image. These variants may include different Kubernetes versions, CNI plugins, " +
+			"or any other variations as needed for testing different Kubernetes features and behaviors.`,",
```
Comment (neolit123) [inline kinder/cmd/kinder/build/build.go:38]:
```suggestion
		Long: "The 'build' command is used for creating images necessary for a Kubernetes cluster created with kinder. \n" +
			"It has two primary subcommands:\n" +
			"1. 'base-image': This command is used to build the base image for nodes in the cluster.\n" +
			"The base image includes all the necessary dependencies and configurations required for Kubernetes nodes.\n" +
			"2. 'node-image-variant': This command is used to build different variants of the node images based\n" +
			"on the base image. These variants may include different Kubernetes versions, CNI plugins,\n" +
			"or any other variations as needed for testing different Kubernetes features and behaviors.",
```

you must have `\n` if not using the quotes `

Comment (chansuke) [reply kinder/cmd/kinder/build/build.go]: @neolit123 Thank you for your review.

File: kinder/cmd/kinder/build/build.go [hunk 8]
```diff
@@ -26,11 +26,16 @@ import (
 // NewCommand returns a new cobra.Command for building
 func NewCommand() *cobra.Command {
 	cmd := &cobra.Command{
-		Args: cobra.NoArgs,
-		// TODO(bentheelder): more detailed usage
+		Args:  cobra.NoArgs,
 		Use:   "build",
 		Short: "Build one of [base-image, node-image-variant]",
-		Long:  "Build the base node image (base-image) or node image variants (node-image-variant)",
+		Long: "The 'build' command is used for creating images necessary for a Kubernetes cluster created with kinder. \n" +
```
Comment (neolit123) [inline kinder/cmd/kinder/build/build.go:32]: remove extra " " space before \n on all the lines

File: kinder/cmd/kinder/build/build.go [hunk 9]
```diff
@@ -26,11 +26,16 @@ import (
...
 func NewCommand() *cobra.Command {
 	cmd := &cobra.Command{
-		Args: cobra.NoArgs,
-		// TODO(bentheelder): more detailed usage
+		Args:  cobra.NoArgs,
 		Use:   "build",
 		Short: "Build one of [base-image, node-image-variant]",
-		Long:  "Build the base node image (base-image) or node image variants (node-image-variant)",
+		Long: "The 'build' command is used for creating images necessary for a Kubernetes cluster created with kinder.\n" +
+			"It has two primary subcommands: \n" +
```
Comment (neolit123) [inline kinder/cmd/kinder/build/build.go:33]:
```suggestion
			"It has two primary subcommands:\n" +
```

**Review by neolit123 — COMMENTED** [review]
/approve

one final problem in the text...

**Review by neolit123 — COMMENTED** [review]
/lgtm

**Pull request merged**